generic/java/egg-java.json

```diff
@@ -25,7 +30,7 @@
     "scripts": {
         "installation": {
             "script": "#!\/bin\/bash\r\n# Java Installation Script\r\n#\r\n# Server Files: \/mnt\/server\r\nPROJECT=java\r\n\r\napt update\r\napt install -y curl jq\r\n\r\ncd \/mnt\/server",
-            "container": "debian:buster-slim",
+            "container": "ghcr.io\/parkervcp\/yolks:debian",
```

Comment:
Please don't use this installer image as it is not updated for arm64.

Reply:
> Please don't use this installer image as it is not updated for arm64.

Thats not the installer ;)

Reply:
ok I am blind :)